week-2/stretch/count-words.js

```diff
@@ -26,3 +26,43 @@
 
 3. Order the results to find out which word is the most common in the chapter
 */
+function countWords(str) {
+  let obj = {};
+   const word = str.replace(/[.,!?]/g,'').toLowerCase();
+   const wordArray= word.split(' ');
+   
+   const setOfWordArray = new Set(wordArray);
+  //  console.log(word, wordArray,setOfWordArray);
+  //  for(const items of setOfWordArray){
+  //   let wordsCount = 0;
+  //   for(const elements of wordArray){
+  //     if(items === elements){
+  //       wordsCount++;
+  //     }
+    
+  //   }
+  //   if(items.length>0){
+  //     obj[items] = wordsCount;
+
+  //   }
+      
+  //  }
+  const arr =[];
+  for(const items of setOfWordArray){
+    
+    if(items.length>0){
+      let key = items;
+      let value = wordArray.filter(element => element === items).length;
+      arr.push([key,value]);
```

Comment:
This code looks good, a few extra comments to make it easier to get at a glance could be good though